HISTORY.rst

```diff
@@ -21,14 +21,6 @@ Unreleased Changes
 * GLOBIO
     * Fix a bug that mishandled combining infrastructure data when only one
       infrastructure data was present.
-* Urban Flood Risk Mitigation
-    * Fixed an issue where the output vector ``flood_risk_service.shp`` would
-      only be created when the built infrastructure vector was provided.  Now,
-      the ``flood_risk_service.shp`` vector is always created, but the fields
-      created differ depending on whether the built infrastructure input is
-      present during the model run.
-    * Fixed an issue where the model would crash if an infrastructure geometry
-      were invalid or absent.  Such features are now skipped.
```

Comment:
Does this PR really remove an update to HISTORY? Just checking in case there was some weird merge thing.